bin/rad-patch

```diff
@@ -189,162 +189,31 @@
 
 ;; showing patches
 
-(def pretty-state
-  (fn [state]
-    (drop 1 (show state))))
-
-(def cells
-  {:state       10
-   :number      10
-   :title       50
-   :author      20
-   :modified-at 22})
-
-(def wrap-cell
-  (fn [phrase cell-key]
-    (def c-length (lookup cell-key cells))
-    (string-append (take c-length (string/pad-right-to c-length phrase)) " ")))
-
-(def pretty-table-header
-  (fn []
-    (string-append
-      (wrap-cell "state" :state)
-      (wrap-cell "#" :number)
-      (wrap-cell "commit" :title)
-      (wrap-cell "author" :author)
-      (wrap-cell "updated" :modified-at))))
-
-(def pretty-modified-at
-  (fn [i]
-    (string-append (lookup :modified-at i)
-                   (if (lookup :read? i) "  " " *"))))
-
-(def pretty-headline
-  (fn [i]
-    (match i
-      '{:number n
-        :title tit
-        :state s
-        :author-name an}
-      (string/unwords
-        [(pretty-state s)
-         (show n)
-         tit
-         an
-         (pretty-modified-at i)]))))
-
-(def pretty-row
-  (fn [i]
-    (match i
-      '{:number n
-        :title tit
-        :state s
-        :author-name an}
-      (string-append
-        (wrap-cell (pretty-state s) :state)
-        (wrap-cell (show n) :number)
-        (wrap-cell tit :title)
-        (wrap-cell an :author)
-        (wrap-cell (pretty-modified-at i) :modified-at)))))
-
-(def pretty-comment
- (fn [c]
-   (match c
-     '{:created-at c-a
-       :body b
-       :author-name a-n}
-     (string-append "### " a-n " [" c-a "]\n\n" b "\n"))))
-
-(def pretty-patch
-  (fn [i]
-    (match i
-      '{:patch p
-        :created-at c-a
-        :comments cs}
-      (string-append
-        (pretty-headline i) "\n\n"
-        p "\n\n"
-        "Comments\n"
-        "--------\n\n"
-        (string/unlines (map pretty-comment cs))))))
-
-(def comment-divider "----------------------------------------")
-
-(def add-comment-md
-  (fn [x]
-    (string-append
-     "\n" comment-divider "\n"
-     "---Add comment above the line.\n\n"
-     x)))
-
 (def add-read-status!
   (fn [machine i]
     (insert :read? (newness/read-since-modified!? machine i) i)))
 
-(def list-items
-  "Lists all patches and filters them by `options`. If `--new` is passed as an
-  option only recent items are shown. If `--fancy is passed, the list is opened
-  via fzf."
-  (fn [machine options]
-
-    (def mk-key
-      (fn [patch]
-        (if (lookup :fancy options)
-          [(pretty-headline patch) patch]
-          [(pretty-row patch) patch])))
+(def add-username
+  (fn [d]
+    (def n (lookup-default :author-name "???" d))
+    (insert :username n d)))
 
-    (def state-filt
-      (match (map (fn [opt] (read (string-append ":" opt))) (lookup :state options))
-        []      (fn [x] #t)
-        'args   (fn [x] (elem? (lookup :state x) args))))
-
-    (def patches (values (list-patches machine)))
+(def add-username-patch
```

Comment:
This function and `add-username-issue` are the same, can we move them in `items.rad` (along with the `add-username` function)?